src/main/java/com/cognologix/BankSystemApplicationAssignment/service/services/AccountServices.java

```diff
@@ -0,0 +1,16 @@
+package com.cognologix.BankSystemApplicationAssignment.service.services;
+
+import com.cognologix.BankSystemApplicationAssignment.dto.AccountDto;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
```

Comment:
service package can have interfaces package and implementation packages and respective interfaces and classes in them

Reply:
yes